Sources/StreamVideo/WebRTC/WebRTCClient.swift

```diff
@@ -1311,9 +1311,10 @@ class WebRTCClient: NSObject, @unchecked Sendable {
     
     private func addOnParticipantsChangeHandler() {
         Task { [weak self] in
```

Comment:
Unrelated but Is it OK that the Task is never cancelled?

Reply:
Definitely! Thanks :)